html/changelog.html

```diff
@@ -55,1593 +56,1386 @@
 		
 		-->
 		<div class="commit sansserif">
-			<h2 class="date">21 April 2018</h2>
-			<h3 class="author">SailorDave updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="rscadd">A new manipulation circuit, the Seed Extractor. Extracts seeds from produce, and outputs a list of the extracted seeds.</li>
-				<li class="rscadd">A new list circuit, the List Filter. Searches through a list for anything matching the desired element and outputs two lists: one containing just the matches, and the other with matches filtered out.</li>
-				<li class="rscadd">A new list circuit, the Set circuit. Removes duplicate entries from a list.</li>
-				<li class="tweak">The Plant Manipulation circuit can now plant seeds, and outputs a list of harvested plants.</li>
-				<li class="tweak">Reagent circuits can now irrigate connected hydroponic trays and inject blood samples into Replica pods.</li>
-				<li class="tweak">The Examiner circuit can now output worn items and other examined details of carbon and silicon mobs into the description pin, as well as the occupied turf of the scanned object, and can now scan turfs.</li>
-				<li class="tweak">The locomotion circuit now allows drones to move at base human speed, but cannot be stacked, and assemblies can now open doors they have access to like bots by bumping into them.</li>
-				<li class="bugfix">List Advanced Locator circuit now accepts refs as well as strings.</li>
-				<li class="bugfix">Fixed the Power Transmitter circuit not properly displaying a message when activated.</li>
-				<li class="bugfix">Medical Analyzer circuit can now properly scan non-human mobs.</li>
+			<h2 class="date">04 September 2018</h2>
+			<h3 class="author">Denton updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="spellcheck">Mech construction messages no longer incorrectly mention high-tier stock parts.</li>
+				<li class="tweak">Added a nanite lab to Deltastation! It's at the old EXPERIMENTOR lab.</li>
+				<li class="tweak">Delta: Moved the Xenobiology disposals bin to be less obstructive. Added two sets of insulated gloves to Engineering.</li>
+				<li class="bugfix">Delta: Fixed scientists not having maintenance access near the circuitry lab and toxins launch chamber.</li>

```

Comment:
We don't have roundstart nanite labs here, changelog is false advertisement

Reply:
considering there is a big sync that is "supposed" to be done, it's one of those things.

Reply:
A soft sync that only includes the backend stuff, yes.

Reply:
unless nanite stuff's been mapped in to all maps, it might be worth considering doing a update for the little things like that.

the code base is hella out of date.